src/routes/support/Contribute.tsx

```diff
@@ -0,0 +1,149 @@
+              <a
+                href="https://goa-dio.atlassian.net/wiki/spaces/DS/pages/2342813697"
+                target="_blank">
+                Design System Drop-in Hours
+              </a>
+              . No need to schedule a time, you can just show up!
+            </div>
+            <strong>Be prepared to:</strong>
+            <ul>
+              <li>Explain your project goals</li>
+              <li>
+                Share what options and iterations were explored with and without the Design System
+                components
+              </li>
+            </ul>
+          </li>
+          <li>
+            We will evaluate the contribution against the design system contribution criteria.
+          </li>{" "}
+          <li>
+            Once approved, the item will be added to the Design System's backlog. As work commences,
+            we will collaborate with the team that made the request to ensure the component meets
+            their specific needs.
+          </li>
+        </ol>
+        <h3>Criteria for being part of the design system</h3>
+        <h4>Useful</h4>
+        <p>
+          There is evidence that this component or pattern would be useful for many teams or
+          services. Evidence could be screenshots or links to versions of it being used in different
+          services.
+        </p>
+        <h4>Unique</h4>
+        <p>
+          It does not replicate something already in the Design System. <br />
+          If it's intended to replace an existing component or pattern, there is evidence to show
+          that it's better than the existing version.
+        </p>
+        <h4>Good user experience</h4>
+        <p>
+          It creates a more accessible, usable, and simpler experience for the user. The user has a
+          greater understanding and consent to what they are doing. <br />
+          It should be informed by research to show that it improves user experience and is
+          responding to a user need.
+        </p>
+        <h4>Universal</h4>
+        <p>
+          It is inclusive and accessible. <br />
+          It should meet or exceed accessible standards. <br />
+          <a href="https://goa-dio.atlassian.net/wiki/spaces/DIO/pages/1702690950" target="_blank">
+            WCAG 2.1
+          </a>
+        </p>
+      </div>
+
+      <GoADivider mb="2xl" mt="2xl"></GoADivider>
+      <div className="max-width-72ch contribute-page-content">
+        <h2 id="development">Code contribution process</h2>
+        <ol>
+          <li>
+            Make sure your component has already gone through the Design contribution process
+            outline above.
+          </li>
+          <li>
+            Follow the <Link to="/get-started/developers/setup">contributor setup process</Link> to
+            make sure that you're setup to contribute to our repo. Make sure you're forking our
+            repo, not directly cloning our repo
+          </li>
+          <li>
```

Comment:
You should change this whole point to reference updating the playground instead of the apps, the apps won't exist soon after we make this site public.

Reply:
It says, "please be sure to update the angular demo (app/angular-demo) and the react demo (/app/react-demo)". What I'm saying is, those apps (angular-demo, react-demo), won't exist after this. The playground in `/libs/web-components/playground` will exist however and should be used for testing.